dsaas-services/rhche-redirector.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+- hash: master
+  name: rhche-redirector
+  path: homeless-templates/rhche-redirector.yaml
+  url: https://github.com/openshiftio/saas-openshiftio/
+  hash_length: 6
```

Comment:
please add new line

Reply:
Why? I don't see the other services yaml files ending in an empty newline either.Why? I don't see the other services yaml files ending in an empty newline either.